include/deal.II/base/quadrature_lib.h

```diff
@@ -47,6 +47,58 @@ class QGauss : public Quadrature<dim>
   QGauss(const unsigned int n);
 };
 
+/**
+ * The Gauss-Radau family of quadrature rules for numerical integration.
```

Comment:
Could you add a sentence about when this quadrature rule is useful.

Reply:
Done